ui/shared/components/panel/search/VariantSearchFormPanels.jsx

```diff
@@ -95,23 +93,20 @@ const LeftAligned = styled.div`
 
 const LazyLabeledSlider = props => <React.Suspense fallback={<Loader />}><LabeledSlider {...props} /></React.Suspense>
 
-const JsonSelectPropsWithAll = (options, all) => ({
+export const JsonSelectPropsWithAll = (options, all) => ({
```

Comment:
The name `JsonSelectPropsWithAll` doesn't make a lot of sense. How about changing to another name like `getPathPanelProps`?

Reply:
this is copy pasted and they are the properties for a select component with json values and an "all" option which is used by several non-pathogenic related components, so I will leave it as is